### PR TITLE
Update add-storage-space.md

### PR DESCRIPTION
**Pull request by msbemba**
Changed navigation institutions as it is no longer available under purchase services

fixes#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/10411

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/commerce/add-storage-space.md [hunk 1]
```diff
@@ -62,12 +62,13 @@ For information about setting site collection storage limits, see [Manage site c
...
-7. On the **Complete order** page, verify the total. If you need to make any changes, select **Edit order**. If the order requires a credit check, select the check box. When you're finished, select **Place order** \> **Go to Admin Home**.
+2. In the **Your Products** page, Select the subscription that contains **Sharepoint online plan 1** or **Sharepoint online plan 2**.
+3. On the **Product details and upgrade** section, select **View apps and services included with this subscription**.
+4. Click on the **Add Ons** tab,  find **Office 365 Extra File Storage**, and select Details.
+5. On the product details page, select Next and **Enter the number of gigabytes of storage** you want to add.
+6. Select **Check out now**.
+7. On the **How does this look?** page, verify the number of gigabytes of storage you selected, review the pricing information, and then select **Next**.
+8. On the **Complete order** page, verify the total. If you need to make any changes, select **Edit order**. If the order requires a credit check, select the check box. When you're finished, select **Place order** \> **Go to Admin Home**.
```
Comment (JohanFreelancer9) [inline microsoft-365/commerce/add-storage-space.md]:
```suggestion
2. On the **Your Products** page, select the subscription that contains **Sharepoint Online Plan 1** or **Sharepoint Online Plan 2**.
3. In the **Product details and upgrade** section, select **View apps and services included with this subscription**.
4. Select the **Add-ons** tab,  find **Office 365 Extra File Storage**, and select **Details**.
5. On the product details page, select **Next** and **Enter the number of gigabytes of storage** you want to add.
6. Select **Check out now**.
7. On the **How does this look?** page, verify the number of gigabytes of storage you selected, review the pricing information, and then select **Next**.
8. On the **Complete order** page, verify the total. If you need to make any changes, select **Edit order**. If the order requires a credit check, select the check box. When you're finished, select **Place order** > **Go to Admin Home**.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@cmcatee-MSFT Please merge this PR. Thanks

**Comment by yogkumgit** [comment]
@cmcatee-MSFT gentle reminder.

**Review by cmcatee-MSFT — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR, as author has approved it. Thanks

**Pull request merged**